docs/api/fallback/function.json

```diff
@@ -10,7 +10,7 @@
 		{
 			"type": "http",
 			"direction": "out",
-			"name": "res"
+			"name": "$return"
```

Comment:
Allows us to use return statements in the Azure Function, rather than setting the `res` value.